pacoloco.go

```diff
@@ -155,6 +159,17 @@ func forceCheckAtServer(fileName string) bool {
 	return false
 }
 
+var (
+	served_cache = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
The names need to follow Golang naming convention i.e. mixedCaps for varibales. Here and below.

https://betterprogramming.pub/naming-conventions-in-go-short-but-descriptive-1fa7c6d2f32a